src/abstractdataframe/abstractdataframe.jl

```diff
@@ -279,22 +281,72 @@ to each column as the only argument. For columns allowing for missing values,
 the vector is wrapped in a call to [`skipmissing`](@ref): custom functions must therefore
 support such objects (and not only vectors), and cannot access missing values.
 
+For consistency with DataAPI.jl an `io` argument can be passed to `describe` in
```

Comment:
Shouldn't we just drop this (in another PR)? Clearly if you pass `io` you expect the result to be printed there, so it's actively misleading not to throw an error. Or the method taking `io` should call `show(io, describe(...))`.

Anyway when moving things from StatsBase to Statistics I'd like to change that API so that `describe` always returns a special object that can be printed, instead of printing the result directly. That's more useful and more standard.

Reply:
100% agreed in all you write - passing `io` made no sense to me here, but it was defined in DataAPI.jl and I did not want to make breaking changes in this PR.

So - after merging this PR do we make another PR that deprecates `io`? (this is what I would do in DataFrames.jl; all other actions are needed in other packages, but they can be done independently, as we clearly document how `describe` with `AbstractDataFrame` works).

Reply:
Right, I think we can start changing this in DataFrames, the StatsBase to Statistics move will take longer.